pysph/examples/rigid_body/sphere_in_vessel_akinci.py

```diff
@@ -0,0 +1,220 @@
+"""A sphere of density 500 falling into a hydrostatic tank (15 minutes)
+
+Check basic equations of SPH to throw a ball inside the vessel
+"""
+from __future__ import print_function
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
Do not do this as our test suite is sometimes run on machines without matplotlib and this import may fail.  No need for this.